debian/changelog

```diff
@@ -1,3 +1,9 @@
+opx-platform-config (1.5.1+opx1) unstable; urgency=medium
```

Comment:
For future reference, there is no need to use the "+opx" suffix for package versions from this repo.  Just bump the patch version number for a minor fix like this, i.e. 1.5.1 -> 1.5.2.